ui/component/or-chart/src/index.ts

```diff
@@ -962,8 +1028,8 @@ export class OrChart extends translate(i18next)(LitElement) {
                 stepSize = 1;
                 break;
             case "week":
-                interval = DatapointInterval.HOUR;
-                stepSize = 6;
+                interval = DatapointInterval.DAY;
+                stepSize = 1;
```

Comment:
Doesn't this result in a single data point per day?

Reply:
Did some testing and this is correct indeed. Reverted it back.